auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -66,7 +66,7 @@
             style="@style/FirebaseUI.Text.BodyText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="336dp"
+            android:layout_marginTop="@dimen/register_field_padding_vert_3x"
```

Comment:
@samtstern Oh crud, I completely forgot about that. ☹️ There's got to be a better way. We're using margins right now so when the hint slides up and expands, it pushes itself down. What if we used raw `paddingTop`? I think that won't push the view down since it should just expand inside the padding. If you don't have the time, I'm happy to play around with it! 😄